core/src/main/scala/kafka/server/metadata/BrokerMetadataListener.scala

```diff
@@ -204,7 +204,7 @@ class BrokerMetadataListener(
       batch.records().forEach { messageAndVersion =>
         if (isTraceEnabled) {
           trace("Metadata batch %d: processing [%d/%d]: %s.".format(batch.lastOffset, index + 1,
-            batch.records().size(), messageAndVersion.message().toString()))
+            batch.records().size(), messageAndVersion.message()))
```

Comment:
I didn't notice this earlier but maybe we can just first this to use Scala's `s""` interpolation instead of `format`.